src/main/kotlin/no/nav/familie/baks/mottak/integrasjoner/BarnetrygdOppgaveMapper.kt

```diff
@@ -15,7 +15,14 @@ class BarnetrygdOppgaveMapper(hentEnhetClient: HentEnhetClient, pdlClient: PdlCl
     // Behandlingstema og behandlingstype settes basert på regelsettet som er dokumentert nederst her: https://confluence.adeo.no/display/TFA/Mottak+av+dokumenter
     override fun hentBehandlingstema(journalpost: Journalpost): Behandlingstema? {
         return when {
-            erEØS(journalpost) -> Behandlingstema.BarnetrygdEØS
+            journalpost.erBarnetrygdSøknad() ->
+                if (utledBehandlingKategoriFraSøknad(journalpost) == BehandlingKategori.EØS) {
```

Comment:
Slik jeg leser utledBehandlingKategoriFraSøknad så vil dette fungere dårlig for EØS som kommer inn gjennom kanal SKAN.